lambdas/stac-gen/handler.py

```diff
@@ -101,6 +101,18 @@ def create_stac_item_with_cmr(event):
     )
     return stac_item
 
+def get_maria_dt(url):
```

Comment:
Can we put this in a separate file since it's specific to the black marble data for Maria and then import it, that way this file stays more so this file is more generically about stac metadata?

... I think in the future we will want to enable something like metadata templating with something like https://mozilla.github.io/nunjucks/ but this is good for now.